overlay/etc/cron.daily/invoiceninja

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-
-php /var/www/invoiceninja/artisan ninja:send-invoices
-php /var/www/invoiceninja/artisan ninja:send-reminders
```

Comment:
I recall that you mentioned that this cron job is no longer relevant as Invoice Ninja now takes care of background tasks itself. But wouldn't it still need something to trigger the jobs to run? I vaguely recall us talking about a more generic cron job that fires every minute?!

Reply:
Yeah, see bottom of ``conf.d/main``. I can change it to an overlay if you'd prefer, but since it's only 1 line I figured better to keep it all together.

Reply:
Ah, sorry I missed that. I'm fine with that, just missed that you had included in conf script.